airbyte-workers/build.gradle

```diff
@@ -37,6 +37,19 @@ dependencies {
     integrationTestJavaImplementation project(':airbyte-workers')
 }
 
+sourceSets {
```

Comment:
woof. i think we should do this in the opposite direction. as it is now, we are, effectively, making the worker depend on normalization. instead, i think it is more palatable to have normalization to depend on the worker. in other words, we prefer for normalization (which looks more like a connector) to depend on some piece of the core platform than the other way around.

They way I would think about doing this is move the `sshtunnel.sh` into workers (in the dir as `dbt_transformations_entrypoint.sh`). Then I would have base-normalization copy that file from airbyte-workers to. You can leverage a gradle `COPY` command like in this [example](https://github.com/airbytehq/airbyte/blob/master/airbyte-server/build.gradle#L87-L96)

That's one suggestion to how to tackle this. I think the key thing is to keep the dependency hierarchy intuitive (even if we have to accept some peculiarities). This yet another reason we need to go back and rethink how normalization fits into our code base. 😅

Reply:
I prefer your approach hierarchically, makes sense.